src/core/text/Text.js

```diff
@@ -246,11 +183,14 @@ Object.defineProperties(Text.prototype, {
 
 /**
  * Renders text and updates it when needed
- *
+ * @param respectDirty {boolean} Whether to abort updating the text if the Text isn't dirty and the function is called.
  * @private
  */
-Text.prototype.updateText = function ()
+Text.prototype.updateText = function (respectDirty)
```

Comment:
Looks like most calls to `updateText` pass `true`. Would it make more sense to have this be something like `skipDirtyCheck` so that the default respects dirty, but you can override that behavior in rare cases?